test-data/unit/check-classes.test

```diff
@@ -2051,6 +2051,16 @@ class X: pass
 foo(X)
 [out]
 
+[case testTypeUsingTypeCTypeAnyMember]
+from typing import Type, Any
+def foo(arg: Type[Any]):
+    x = arg.member_name
```

Comment:
Thank you @ilevkivskyi , using that I just updated the PR and now it handles fallback to builtins.type attribute if the attribute is present there